docs/getting_started/installation.md

```diff
@@ -545,10 +540,10 @@ user@hostname:~/irods/build $ make non-package-install-postgres
 The iCommands are a dependency of the iRODS server, and can be built with the following sequence:
 
 ~~~
-user@hostname:~/irods_client_icommands/build $ cmake -DCMAKE_INSTALL_PREFIX=$IRODS_INSTALL_DIR -DIRODS_DIR=$IRODS_INSTALL_DIR/usr/lib/irods/cmake ../
+user@hostname:~/irods_client_icommands/build $ cmake -DCMAKE_INSTALL_PREFIX=$IRODS_INSTALL_DIR -DIRODS_DIR=$IRODS_INSTALL_DIR/lib/irods/cmake -DBUILD_DOCS=NO ../
 user@hostname:~/irods_client_icommands/build $ make install
-user@hostname:~/irods_client_icommands/build $ export LD_LIBRARY_PATH=$IRODS_INSTALL_DIR/usr/lib
-user@hostname:~/irods_client_icommands/build $ export PATH=$IRODS_INSTALL_DIR/usr/bin:$IRODS_INSTALL_DIR/usr/sbin:$PATH
+user@hostname:~/irods_client_icommands/build $ export LD_LIBRARY_PATH=$IRODS_INSTALL_DIR/lib
```

Comment:
Add line:
```bash
export LD_LIBRARY_PATH=$IRODS_EXTERNALS/boost1.78.0-0/lib:$LD_LIBRARY_PATH
```

Reply:
or do it in one?

```
export LD_LIBRARY_PATH=$IRODS_INSTALL_DIR/lib:$IRODS_EXTERNALS/boost1.78.0-0/lib:$LD_LIBRARY_PATH
```

Reply:
I didn't think too hard on the multi-line vs single line. I landed on the multi-line one only because it is easier to see what is happening.

Thinking about it more, combining the lines removes one step. The reader can take time to parse the line to see what it's doing if they feel the need to.

Reply:
yes, one seems best.

Reply:
done